### PR TITLE
Release v1.30.0

### PR DESCRIPTION
**Pull request by tejal29**
Should merge #6410
release notes for v1.30.0

### CODE REVIEW

**Comment by tejal29** [comment]
See https://github.com/GoogleContainerTools/skaffold/pull/6410 for test failure.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
+# v1.30.0 Release - 08/11/2021
+Note: This release comes with a new config version, `v2beta21`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Support deploy hooks for Skaffold lifecycle hooks [#6376](https://github.com/GoogleContainerTools/skaffold/pull/6376)
```
Comment (gsquared94) [inline CHANGELOG.md:5]: can you link the docs? https://skaffold.dev/docs/pipeline-stages/lifecycle-hooks/

**Comment by gsquared94** [comment]
the release commit needs to be:
```
release: v1.30.0
```

Otherwise the GA won't be able to parse the tag and eventually fail.

File: CHANGELOG.md (hunk 1 above)
Comment (tejal29) [reply CHANGELOG.md]: done!!

**Comment by tejal29** [comment]
> the release commit needs to be:
> 
> ```
> release: v1.30.0
> ```
> 
> Otherwise the GA won't be able to parse the tag and eventually fail.

I read this while listening to the song https://youtu.be/YR12Z8f1Dh8?t=20

Exactly my question

**Review by gsquared94 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+* hooks: update deploy docs [#6386](https://github.com/GoogleContainerTools/skaffold/pull/6386)
+* [design proposal] Add config option 'deploy.config.transformableAllowList' [#6236](https://github.com/GoogleContainerTools/skaffold/pull/6236)
+* GitLab (with capital 'L') [#6384](https://github.com/GoogleContainerTools/skaffold/pull/6384)
+* Validate generated schemas in generator script [#6385](https://github.com/GoogleContainerTools/skaffold/pull/6385)
+* fix: schema gen for `ContainerHook` [#6372](https://github.com/GoogleContainerTools/skaffold/pull/6372)
+* Add trace-level port-allocation logs [#6293](https://github.com/GoogleContainerTools/skaffold/pull/6293)
+* Bump cloud.google.com/go/storage from 1.10.0 to 1.16.0 [#6324](https://github.com/GoogleContainerTools/skaffold/pull/6324)
+* Mark GCP Buildpacks builder as trusted in the examples [#6284](https://github.com/GoogleContainerTools/skaffold/pull/6284)
+
+
```
Comment (aaron-prindle) [inline CHANGELOG.md:38]: nit: did we want to add back the install commands (`sudo mv skaffold /usr/local/bin ...`)

**Review by aaron-prindle — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (tejal29) [reply CHANGELOG.md:50]: done! Thanks!

**Pull request merged**